molo/surveys/templates/surveys/molo_survey_page.html

```diff
@@ -7,7 +7,7 @@
 <div class="surveys surveys{{page.get_parent_section.get_effective_extra_style_hints}}">
   <h1 class="surveys__title">{{ page.title }}</h1>
   <h4 class="surveys__intro">{{ page.intro }}</h4>
-  {% if page.terms_and_conditions.first.exists %}
+  {% if page.terms_and_conditions.exists and page.terms_and_conditions.first.terms_and_conditions.live %}
```

Comment:
I don't understand the section part.. why not `page.terms_and_conditions.first.live`?

Reply:
@miltontony because page.terms_and_conditions.first is a relation not the actual page so it does not have the live attribute

Reply:
we face this problem because there is no singular form of terms_and_conditions, like pants :/